languages/rust/oso/src/builtins.rs

```diff
@@ -29,7 +29,114 @@ fn dictionary() -> Class<HashMap<Symbol, Value>> {
 fn string() -> Class<String> {
     Class::<String>::with_default()
         .name("String")
+        .add_method("len", |s: &String| s.len() as i64)
+        .add_method("is_empty", |s: &String| s.is_empty())
```

Comment:
These would be better as `String::is_empty`, etc.

Reply:
Oh, but its implemented on `str`.

Reply:
I think this is actually the only one that can be written that way.